opencog/cogserver/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Step to take
+# 1. docker build --no-cache -t opencog/cogserver .
+# 2. docker run -p 17001:17001 -it opencog/cogserver
+# 3. rlwrap telnet localhost 17001
+
+FROM opencog/opencog-deps
+
+# Clone opencog and build it
+RUN git clone --depth 1 https://github.com/opencog/opencog oc
+RUN cd oc; mkdir build; cd build; cmake ..; make -j$(nproc)
```

Comment:
Ah, OK, that makes sense.   Maybe you should add a note to the base, explaining this (specifically, the bit about the mount)